<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientOnlyView</class>
 <widget class="QMainWindow" name="ClientOnlyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_23">
    <item>
     <widget class="QTabWidget" name="tabView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>-2</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>View Animals</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QTableWidget" name="animalTbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>95</number>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Species</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Breed</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date of Birth</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Age(Y/M)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vaccinated</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>See Detailed Animal View</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Add Client</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>861</width>
          <height>270</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="fNameLbl">
               <property name="text">
                <string>*First Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="firstName">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="lNamelbl">
               <property name="text">
                <string>*Last Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lastName">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="emailLbl">
               <property name="text">
                <string>*Email: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="emailTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="userLbl">
               <property name="text">
                <string>*Username: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="user">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="passLbl">
               <property name="text">
                <string>*Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="confPassLbl">
               <property name="text">
                <string>*Confirm Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="confirmPasswordTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Passwords can include letters and numbers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="passConLbl">
             <property name="text">
              <string>Please ensure both passwords match</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="postLbl">
               <property name="text">
                <string>*Postal Code</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="postalCodeTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="cityLbl">
               <property name="text">
                <string>*City:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cityTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="provLbl">
               <property name="text">
                <string>*Province:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="provinceCombo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="addLn1lbl">
               <property name="text">
                <string>*Address Line 1:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addressLine1Txt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="addLn2Lbl">
               <property name="text">
                <string>Address Line 2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addressLine2Txt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="phoneLbl">
               <property name="text">
                <string>*Phone Number (xxx)-xxx-xxxx:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="phoneTxt"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="emptyClientLbl">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>430</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Items marked with a * are required</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Child Friendliness</string>
        </property>
        <widget class="QRadioButton" name="childFriendly">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>171</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Must be child friendly</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="noChildPreference">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>161</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>No preference</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="notFriendlyChild">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>171</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Not child friendly</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="editClientBtn">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>430</y>
          <width>358</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>280</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Stranger Friendliness</string>
        </property>
        <widget class="QRadioButton" name="strangeFriend">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>171</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Must be stranger friendly</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="noStrangePreference">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>161</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>No preference</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="notFriendlyStrange">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>171</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Not stranger friendly</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>280</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Desired Attributes in a pet:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>400</y>
          <width>331</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>1 being more aggressive, 10 being more passive</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>298</y>
          <width>327</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string>Sociability:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="tempRank">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No preference</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>Temperament:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="socRank">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No preference</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="trainRank">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No preference</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>Ability to train::</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="trainSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="socSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="tempSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
